.circleci/config.yml

```diff
@@ -69,15 +69,35 @@ jobs:
     docker:
       - image: circleci/python:3.6.5-jessie
 
+  publish:
+    docker:
+      - image: circleci/python:3.6.5-jessie
+    steps:
+      - checkout
+      - run: sudo pip install twine
+      - run: make release
+
 workflows:
   version: 2
-  build_and_test:
+  build_test_release:
     jobs:
       - build
       - test_27
       - test_34
       - test_35
       - test_36
+      - publish:
+          requires:
+            - build
```

Comment:
Not a blocker, but instead of listing all the test jobs one by one and duplicating them throughout this file, maybe we could setup a single `test` job that requires all the other `test_xx` to run, and then anything that needs to depend on all of those can just depend on `test`?

Reply:
this has been discarded, it's now done differently. Sorry I was force-pushing, didn't know you were reviewing already.

Reply:
I don't think that's possible, as it seems in CircleCI job dependencies can only between jobs in the same workflow. So even if there's a ```test``` job that requires all other tests, that job has to be specified in all workflows - similar repetition as we see here.

But I'd be happy to be proven wrong, as this looks ugly indeed :)